src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -453,7 +474,8 @@ void func_8092D4D8(EnDns* this, GlobalContext* globalCtx) {
 }
 
 void func_8092D5E8(EnDns* this, GlobalContext* globalCtx) {
-    static s32 D_8092DE0C[] = { 0, 0, 4, 6 };
+    static s32 D_8092DE0C[] = { EN_DNS_ANIMATION_IDLE_1, EN_DNS_ANIMATION_IDLE_1, EN_DNS_ANIMATION_SURPRISE_START,
+                                EN_DNS_ANIMATION_RUN_START };
```

Comment:
```suggestion
    static s32 D_8092DE0C[] = { EN_DNS_ANIMATION_IDLE_1, EN_DNS_ANIMATION_IDLE_1, EN_DNS_ANIMATION_SURPRISE_START,
                                EN_DNS_ANIMATION_RUN_START, };
```
With the enum now in place, this would format better with a trailing comma.